src/System.Runtime.Serialization.Xml/tests/Configurations.props

```diff
@@ -5,6 +5,7 @@
       netstandard;
       netcoreapp;
       uap;
+      netfx;
```

Comment:
We shouldn't add `netfx` in this file. This would cause the test project to build against `netfx` assemblies instead of `netstandard` when targeting `netfx`. We shouldn't even have `netcoreapp` in this file. I recently made a PR, https://github.com/dotnet/corefx/pull/18991, for fixing that.

I think we'd better to use a separate project for canonicalization tests. Can you please create a new folder named `canonicalization` and put a new csproj in it?